packages/embeds/embed-core/playwright/tests/action-based.e2e.ts

```diff
@@ -72,7 +69,7 @@ test.describe("Popup Tests", () => {
     if (!embedIframe) {
       throw new Error("Embed iframe not found");
     }
-    const { uid: bookingId } = await bookFirstEvent("free", embedIframe, page);
+    const { uid: bookingId } = await bookNthEvent("free", embedIframe, page, 2);
```

Comment:
Why do you need to change the event number being used? relying on event index per test this way is bound to cause issues as someone can start using the same index in another test.
We should stick to booking a default event(the first one) unless the test explicitly needs to test any other event.

Reply:
The problem I faced is with race conditions when booking at the same time with different tests. I wasn't able to find any other case in your tests where this was the case, so I wasn't sure on how to handle it.

The solution I came up with is hardcoding the index of the day to be booked, so that each test can book a different day.

This is why I also deleted the cleanup `afterEach`.

Reply:
Right now there is a test that is failing because of this. Both `"should open embed iframe on click - Configured with light theme"` and `"should be able to reschedule"` use the same `"/free"` account and the same slot.

The second one (`"should be able to reschedule"`) always fails because it is not able to place booking for that slot.

> This is currently happing in the main branch (not in this PR).